compose/compose_strategy.go

```diff
@@ -68,26 +68,6 @@ func NewOAuth2JWTECDSAStrategy(key *ecdsa.PrivateKey, strategy *oauth2.HMACSHASt
 	}
 }
 
-func NewOAuth2JWTStrategyWithIssuer(key *rsa.PrivateKey, strategy *oauth2.HMACSHAStrategy, issuer string) *oauth2.DefaultJWTStrategy {
```

Comment:
I removed this because I realized this can be simply `NewOAuth2JWTStrategy(...).WithIssuer(...)`.

Reply:
I mean, I added this very recently (like few weeks ago). Not sure if anyone else managed to use it in meantime really. So I think it might be worthwhile to make this cleanup.

Reply:
It was added 3 months ago (https://github.com/ory/fosite/commit/901e206d03b615c189e12f94607d92c10d6909fa) and spans 0.32-0.34 so I would keep it, but you can use godoc to mark it as deprecated, and then we can remove it in the next or the version after!